README.md

```diff
@@ -92,6 +92,64 @@ plugins:
   - mkdocstrings
 ```
 
+### Including API Documentation in Navigation
+
+To include the API documentation created by the plugin in your site's
+navigation, you can add the following configuration to your `mkdocs.yml` file:
+
+```yaml
+nav:
+  - ... other navigation sections ...
+  - API Reference: autoapi/
+  - ... other navigation sections ...
+```
+
+More information on navigation configuration can be found in the
+[MkDocs User Guide](https://www.mkdocs.org/user-guide/configuration/#nav).
+
+### Putting It All Together
+
+Again, consider the following project structure:
```

Comment:
May be worth explicitly showing an example project using [src layout](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/) or just mention it in https://github.com/jcayers20/mkdocs-autoapi/tree/feature-document-nav?tab=readme-ov-file#setting-the-project-root:

```yaml
plugins:
  - mkdocs-autoapi:
      project_root: src/
```